submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/FileApiTest.kt

```diff
@@ -40,8 +41,8 @@ internal class FileApiTest(private val tempFolder: TemporaryFolder) : BaseIntegr
 
         @BeforeAll
         fun init() {
-            securityService.registerUser(SignUpRequest("test@biostudies.com", "jhon_doe", "12345"))
-            webClient = BioWebClient.create("http://localhost:$serverPort", securityService.login("jhon_doe", "12345"))
+            securityService.registerUser(RegisterRequest("test@biostudies.com", "jhon_doe", "12345"))
+            webClient = SecurityWebClient.create("http://localhost:$serverPort").auhtenticate("jhon_doe", "12345")
```

Comment:
The `authenticate` method makes it look like a single authentication request instead of a method that uses those credentials to authenticate and then gives me an authenticated client that we can use.

I would suggest changing the name to something like `getAuthenticatedClient` or `getAuthenticatedBioWebClient`. It's a bit longer but it makes clear what we're trying to do.